### PR TITLE
Add retrieving occurences data from obis

### PR DESCRIPTION
**Pull request by Marie59**
New tool to retrieve occurences data from Obis which can be chained with obisindicators tool. It relies on the package robis

### CODE REVIEW

File: tools/obisindicators/robis.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            '$lat_min'
+            '$lat_max'
+            '$long_min'
+            '$long_max'
+            '$output'
+        ]]>
+    </command>
+    <inputs>
+        <param name="species" type="text" label="Scientific name of the species" help="Genus species format, eg : Scomber scombrus"/>
+        <param name="taxon" type="text" label="Taxon ID"/>
```
Comment (bgruening) [inline tools/obisindicators/robis.xml:27]: should we restrict this text filed a bit? Ony letters? only letter and numbers?

Comment (Marie59) [reply tools/obisindicators/robis.xml:27]: Yes that's a good idea !

Comment (Marie59) [reply tools/obisindicators/robis.xml:27]: Should I use a sanitizer like the following for species :
`            <sanitizer invalid_char="">
                <valid initial="string.ascii_letters">
                </valid>
            </sanitizer>`

Comment (bgruening) [reply tools/obisindicators/robis.xml:27]: Mh I guess a validator is needed

Comment (Marie59) [reply tools/obisindicators/robis.xml:28]: Ah thanks !!

**Comment by Marie59** [comment]
uh weird it does not like the validators ...

File: tools/obisindicators/robis.xml (hunk 1 above)
Comment (bgruening) [reply tools/obisindicators/robis.xml:29]: https://docs.galaxyproject.org/en/latest/dev/schema.html#id54

**Comment by Marie59** [comment]
Youhou !! Thanks @bgruening  !!!

**Pull request merged**